portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -19,14 +19,20 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.*;
+import com.google.gson.*;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+    response.setContentType("application/json;");
+    //response.getWriter().println("Hello Zoe!");
+    response.getWriter().println(new Gson().toJson(values));
   }
+
+  ArrayList<String> values = new ArrayList<>(Arrays.asList("hello", "there", "world"));
```

Comment:
This is outside of the function but I don't see it being used elsewhere.

Maybe inline this into line 33 like so:
ArrayList<String> values = new ArrayList<>(Arrays.asList("hello", "there", "world"));
response.getWriter().println(new Gson().toJson(values))